### ISSUE TITLE
partial hour timezone offset not correctly correlated

### ISSUE THREAD

**Issue by jtmoon79** [jtmoon79/super-speedy-syslog-searcher#25]
Passing a "partial hour" timezone offset, e.g. `+05:30`, is not interpreted to UTC correctly.

```lang-text
$ s4 /var/log -u -a "$(date -d "2 days ago 12:00" '+%Y-%m-%dT%H:%M:%S') +05:30" -b "$(date -d "2 days ago 12:05" '+%Y-%m-%dT%H:%M:%S') +05:30"
20220915T063000.000000 +0000:Sep 14 22:30:00 ubuntu22-s4b sshd[1426912]: Received disconnect from 187.35.147.87 port 56531:11: Bye Bye [preauth]
```

Notice the first `-a` datetime should be a `:30` minute mark. Yet it appears the time `Sep 14 22:30:00` is associated with `-u` datetime `20220915T063000.000000 +0000`, which has the _same_ `:30` minute offset. These times should be 30 minutes different.

**Comment by jtmoon79** [comment]
First thing to check is parsing the passed `-a`; are the minutes getting dropped?

**Comment by jtmoon79** [comment]
Fixed by a71c5e81761deb547c315296004167e13f82fe9b
